content/SCALE/SCALETutorials/SystemSettings/Services/SMBServiceSCALE.md

```diff
@@ -27,18 +27,42 @@ Enter any alias name or names that do not exceed 15 characters in the **NetBIOS
+If using SMB1 clients, select **Enable SMB1 support** to allow legacy SMB1 clients to connect to the server. Note: SMB1 is deprecated. We advise upgrading clients to operating system versions that support modern SMB protocol versions.
 
-If you plan to use the insecure and vulnerable NTLMv1 encryption, select **NTLMv1 Auth** to allow [smbd](https://www.samba.org/samba/docs/current/man-html/smbd.8.html) attempts to authenticate users. This setting enables backward compatibility with older versions of Windows, but we don't recommend it. Do not use on untrusted networks.
+If you plan to use the insecure and vulnerable NTLMv1 encryption, select **NTLMv1 Auth** to allow [smbd](https://www.samba.org/samba/docs/current/man-html/smbd.8.html) attempts to authenticate users.
+This setting enables backward compatibility with older versions of Windows, but we don't recommend it. Do not use on untrusted networks.
 
-Enter any notes about the service configuration in **Description**
+Enter any notes about the service configuration in **Description**.
 
 For more advanced settings, see [SMB Services Screen]({{< relref "SMBServicesScreen.md" >}}).
 
 Click **Save**.
 
 Start the **SMB** service.
 
+### Configuring Transport Encryption
+TrueNAS and Samba default behavior for SMB transport encryption allows SMB clients to negotiate different encryption levels for shares.
+This default setting enables negotiating encryption but does not turn on data encryption globally per share.
+SMB1 and SMB2 provide different settings to change the level of global or per-share SMB encryption applied to connections.
+See [Samba Server SMB Encrypt(s)](https://www.samba.org/samba/docs/current/man-html/smb.conf.5.html#SERVERSMBENCRYPT) for more information.
+
+You can change the SMB service to apply different SMB transport encryption levels to suit your use case. Go to the **SMB** service, found on the **System Settings > Services** screen, and click on **Advanced Settings**.
+
+Click in the **Transport Encryption Behavior** field to select the option and behavior you want applied:
+
+* **Default - follow upstream/TrueNAS default**
+* **Negotiate - only encrypt transport if explicitly requested by the SMB client**
+* **Desired - encrypt transport if supported by client during session negotiation**
+* **Required - always encrypt transport (rejecting access if client does not support encryption - incompatible with SMB1 server `enable_smb1`)**
+
+Select the **Default** option to use the TrueNAS current behavior.
+There is not a technical limitation preventing an SMB client from negotiating an encrypted session if it is required.
+
+If concerned about having Windows SMB clients always using signing in your environment, make a GPO change on the client side to always sign SMB2+ traffic.
```

Comment:
"always using signing" reads a little awkwardly to me. Would it be accurate to change this to "Windows SMB clients always signing" or "Windows SMB clients signing SMB2+ traffic"

Reply:
Neither of your examples is correct, as far as I read them. The concern Andrew addressed was for users concerned about always having to use Windows signing. I'll reword this.

Reply:
I've changed this in the file.